### DISCUSSION TITLE
Layout on android phone using app

### DISCUSSION THREAD

**Discussion by Editpo** [rern/rAudio#1349] [Bug Reports]
Using latest update.
Layout on android phone using the android app is a bit messed up. Can’t get to menu. No play buttons etc

was all ok before update

**Comment by Editpo** [comment]
Layout still messed up on 0616.
No bottom bar, no top bar, no menu.

**Reply by csch0en** [reply]
Hi. I had the same problems on rAudio Android. After playing around with the Web GUI on my in my webbrowser i found my solution. Go in the upper-right corner (gearwheel), than "Playback" and choose "Bars always on". After some minutes and refreshing the GUI on Android the Top and bottom bars are back. Only the portrait/landscape changing does not realy work properly. Maybe this will help you.

**Reply by rern** [reply]
On small screen, top and bottom bars are hidden by default.

Try tapping at the top of Cover art to show an overlay of every button. The top and bottom bar can be hidden for a much cleaner layout. It might be preferable for hidden Time and Volume as well.

**Comment by Editpo** [comment]
@rern I didn’t know that!

**Comment by gsalinaslopez** [comment]
Hi @rern The project looks awesome!
Regarding the Android app, I couldn't find the repo under your account. Is the app open source as well? If not, do you plan to open-source it?
Thanks!

**Comment by williemctell** [comment]
The app is no longer available to me in the Google Play store.  It says it's incompatible with my Android 12 phone and tablet.

**Reply by rern** [reply]
Google has just mandated all apps to be at least for Android 13.

**Reply by klassikmann** [reply]
@rern 
IMHO I assume Google mandated that all apps have to be compatible with Android 13 but not incompatible with all Android < version 13.

**Comment by klassikmann** [comment]
@williemctell 
Mysterious. Yesterday I got on my Android 7 phone an update for the rAudio app (Google Play Store). https://play.google.com/store/apps/details?id=com.raudio says under "News": _Upgrade to Android 12 (mandated by Google Play)_
Today the PlayStore shows on the phone a non-compatibilty notice, but the app works. My Android 7 tablet didn't get the update yesterday and prohibiting now a manual  update.

**Reply by klassikmann** [reply]
@williemctell 
Addendum: Until @rern and/or Google clarify the situation I recommend to use in the meantime your favourite browser accessing `http://<IP-ADDRESS of rAudio>`.

**Comment by rern** [comment]
![image](https://github.com/rern/rAudio/assets/10094236/aff84747-b369-42bd-9251-9098820e1d0a)

**Reply by guerman5** [reply]
@rern 
Is the source code available for open? could you share it?  Have you thought using Fdroid repository

**Reply by rern** [reply]
[rAudio-app](https://github.com/rern/rAudio-app)

**Reply by klassikmann** [reply]
I get a 404.

**Reply by rern** [reply]
Just switched it to public. Try again.

**Comment by klassikmann** [comment]
@rern 
Please have a look into https://developer.android.com/google/play/requirements/target-sdk

> Every new Android version introduces changes that bring security and performance improvements and enhance the Android user experience. Some of these changes only apply to apps that explicitly declare support through their targetSdkVersion manifest attribute (also known as the target API level).
> 
> Configuring your app to target a recent API level ensures that users can benefit from these improvements, **while your app can still run on older Android versions**. 

Today, the rAudio app version 20230828 is the sole app on my Android 9 phone with a statement in the Play Store marking an incompatability to my phone, regardless the fact that the app is working as desired, but not eligible for further updates.

=> Something could be wrong in your last app update.

**Reply by rern** [reply]
In the waring:
- Target API level

In Google Play Console:
- API levels
- Target SDK

In Android Studio:
- Target SDK version
- Min SDK version

Tried:
- Upgrade only Target SDK version - The warning was still there
- Upgrade both Target and Min SDK version - The warning stop

Please tell which could be wrong.

**Reply by klassikmann** [reply]
@rern 
I am of limited help here as I have never dealt with Android app development before. 
I'm guessing that you have to define both Target and Min SDK version as rcommended in
https://support.google.com/googleplay/android-developer/answer/11926878

> Configuring your app to target a recent API level ensures that users benefit from security, privacy, and performance improvements, while still allowing an app to run on older Android versions (down to the specified minSdkVersion). 

I can confirm that the app runs on Android 7, 8, 9 without problems.
